spec/helpers/application_helper_spec.rb

```diff
@@ -76,7 +76,7 @@
       allow_any_instance_of(Project).to receive(:avatar_in_git).and_return(true)
 
       expect(project_icon("#{project.namespace.to_param}/#{project.to_param}").to_s).to match(
-        image_tag(namespace_project_avatar_path(project.namespace, project)))
+        image_tag("http://localhost"+namespace_project_avatar_path(project.namespace, project)))
```

Comment:
Line is too long. [96/80]<br>Surrounding space missing for operator '+'.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.